data/json/recipes/recipe_appliance.json

```
@@ -106,6 +106,38 @@
     "using": [ [ "soldering_standard", 5 ] ],
     "qualities": [ { "id": "SCREW", "level": 1 } ],
     "proficiencies": [ { "proficiency": "prof_elec_soldering" } ],
-    "components": [ [ [ "frame_wood", 1 ] ], [ [ "battery_charger", 1 ] ] ]
+    "components": [ [ [ "frame_wood", 1 ] ], [ [ "battery_charger", 1 ] ], [ [ "medium_battery_cell", 1 ] ] ]
+  },
+  {
+    "result": "portable_air_conditioner",
+    "type": "recipe",
+    "activity_level": "MODERATE_EXERCISE",
+    "reversible": true,
```

With `"reversible": true`, disassembling this AC will take the same 8 hours as it takes to craft it. I suppose disassembling it should require much less time, like 30 minutes or less.
```suggestion
    "reversible": { "time": "30 m" },
```

---

Honestly, as I was just scavenging other appliances json, I had no clear idea of what this does exactly. Your suggestion makes sense and I agree with that.